src/Api/Auth/Models/Public/Request/AssociationWithPermissionsRequestModel.cs

```diff
@@ -9,7 +9,8 @@ public CollectionAccessSelection ToSelectionReadOnly()
         return new CollectionAccessSelection
```

Comment:
This method is called `ToSelectionReadonly`, but it returns a `CollectionAccessSelection`. Can you please update the static method name, i.e. `ToCollectionAccessSelection`.